src/frontend/efiling-frontend/src/components/page/home/__snapshots__/Home.test.js.snap

```diff
@@ -360,8 +362,10 @@ exports[`Home Component matches the snapshot when user cso account exists 1`] =
       >
         <button
           class="navbar-toggler"
+          data-marging-right="undefined"
           data-target="#footerBar"
           data-toggle="collapse"
+          style=""
```

Comment:
if this is empty is it required? non blocker

Reply:
I guess the snapshot just added it in... I am not setting any style on my own and also it is not a property on the footer component. shouldnt be an issue, not 100% sure how to remove this - just auto added